lib/fika/env.ex

```diff
@@ -25,7 +25,10 @@ defmodule Fika.Env do
         "kernel./(Int,Int)" => "Float",
         "kernel./(Int,Float)" => "Float",
         "kernel./(Float,Int)" => "Float",
-        "kernel./(Float,Float)" => "Float"
+        "kernel./(Float,Float)" => "Float",
+        "kernel.||(Bool,Bool)" => "Bool",
+        "kernel.&&(Bool,Bool)" => "Bool",
+        "kernel.!(Bool)" => "Bool"
```

Comment:
Maybe we could think about how to improve this soon. Also, is there any reason on why `if/else` is not a function itself like in elixir?

Reply:
Yeah, we need a proper stdlib

Reply:
Yeah, this is an ugly hack. This will go away as soon after we focus our attention on the compiler state - there's work planned here. Btw, in Elixir, if-else is a macro, not a function, and that expands to a case expression. Fika also converts this into a case exp in ErlTranslate.